src/cogent3/app/io.py

```diff
@@ -150,6 +150,10 @@ class load_aligned(_seq_loader, ComposableAligned):
 
     klass = ArrayAlignment
 
+    _input_types = None
```

Comment:
these attributes are meant to be `frozenset` instances, not raw strings, so that we can apply set operations in evaluating app compatability. That's why the function in `__init__` is wrong